config/routes.rb

```diff
@@ -3,6 +3,7 @@
   root 'welcome#index'
 
   get 'about' => 'welcome#about', as: :about
+  get 'admin_page' => 'welcome#admin_page', as: :admin_page
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.